client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/DistanceHelper.java

```diff
@@ -46,4 +48,19 @@ public static long getLeftAlignmentDistance(String a, String b) {
         return dist;
     }
 
+    /**
+     * Computes a distance to a==b
+     *
+     * @param a
+     * @param b
+     * @return
+     */
+    public static double getDistanceToEquality(long a, long b) {
+        return Truthness.normalizeValue(Math.abs(a - b));
+    }
```

Comment:
when having "distances" to minimize to 0, there is no need to normalize them, unless when we combine them with &&, eg dist(x)+dist(y)

furthermore `a-b` is unfortunately incorrect for `long` and `double` values, as the distance can overflow, ie the distance itself can be greater than `Long.MAX` if `a` is positive and `b` is negative. They could be casted to something like Big values (or whatever it is called), and then distance could be replaced with `Long.MAX` if greater than it

Reply:
ok. I chose to use Math.subtractExact() to handle the special case for overflows